src/sentry/event_manager.py

```diff
@@ -2437,13 +2437,11 @@ def _get_severity_score(event: Event) -> tuple[float, str]:
                 response = severity_connection_pool.urlopen(
                     "POST",
                     "/v0/issues/severity-score",
-                    body=json.dumps_experimental("event-manager.enable-orjson", payload),
+                    body=orjson.dumps(payload).decode(),
```

Comment:
Passing bytes to body is faster:

```suggestion
                    body=orjson.dumps(payload),
```